### PR TITLE
refactor: create a constant and move a method at AnkiServer

### PR DESCRIPTION
**Pull request by BrayanDSO**
Part of #14949

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt [hunk 1]
```diff
@@ -133,6 +128,13 @@ open class AnkiServer(
             }
         }
 
+        fun getSessionBytes(session: IHTTPSession): ByteArray {
+            val contentLength = session.headers["content-length"]!!.toInt()
+            val bytes = ByteArray(contentLength)
+            session.inputStream.read(bytes, 0, contentLength)
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt:134]: I slightly feel this should be refactored to `session.inputStream.toByteArray(length)`

File: AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt [hunk 2]
```diff
@@ -42,10 +42,10 @@ private const val PORT = 0
 
 open class AnkiServer(
     val activity: FragmentActivity
-) : NanoHTTPD("127.0.0.1", PORT) {
+) : NanoHTTPD(LOCALHOST, PORT) {
 
     fun baseUrl(): String {
-        return "http://127.0.0.1:$listeningPort/"
+        return "http://$LOCALHOST:$listeningPort/"
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt:48]: Agreed: I feel `LOCALHOST` is better than `this.hostname`

File: AnkiDroid/src/main/java/com/ichi2/anki/ReviewerServer.kt [hunk 3]
```diff
@@ -29,7 +29,7 @@ class ReviewerServer(activity: AbstractFlashcardViewer, private val mediaDir: St
 
     override fun start() {
         super.start()
-        Timber.i("Starting server on http://127.0.0.1:$listeningPort")
+        Timber.i("Starting server on http://$LOCALHOST:$listeningPort")
```
Comment (david-allison) [inline AnkiDroid/src/main/java/com/ichi2/anki/ReviewerServer.kt:32]: I'd consider `baseUrl()` here

**Review by david-allison — APPROVED** [review]
Only nits, implementer's choice

File: AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt [hunk 4]
```diff
@@ -119,6 +119,8 @@ open class AnkiServer(
     }
 
     companion object {
+        const val LOCALHOST = "127.0.0.1"
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/pages/AnkiServer.kt:115]: image

**Review by mikehardy — APPROVED** [review]
going with it as is for velocity

**Pull request merged**